content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: blog
+title: "Time to Upgrade; Migrating to Gateway API"
+date: 2023-10-24T10:00:00-08:00
+slug: introducing-ingress2gateway
+---
+
+***Authors:*** Lior Lieberman (Google), Kobi Levi (independent)
+
+Today we are releasing [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway), a tool
+that can help you migrate from Ingress to Gateway API. Gateway API is just weeks away from
+graduating to GA, if you haven't upgraded yet, now's the time to think about it!
```

Comment:
Ideally: explain the “GA” abbreviation

Reply:
like "GA (General Availability)" or did you think of something more than that?

Reply:
“GA (General Availability)” is fine